CHANGELOG.md

```diff
@@ -51,6 +52,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix triggering `ORDER_CANCELED` event at the end of transaction (#10242) (d9eecb2ca)
 - Fix post-migrate called for each app module (#10252) (60205eb56)
 - Only handle known URLs (disable appending slash to URLs automatically) - #10290 by @patrys
+- Introduce tax configuration base on channel in the Avatax plugin - #10445 by mociepka
```

Comment:
@mociepka Please just move this entry to the "3.7.0 Unreleased" section, as we're going to release it as 3.7.

Reply:
Done.